modules/ROOT/pages/managing_extension_studio.adoc

```diff
@@ -0,0 +1,51 @@
+= Managing Extensions in the Studio
+
+:description: The Bonita platform is highly extensible so we created a dedicated view to manage the extensions of your project
+
+The Bonita platform is highly extensible so we created a dedicated view to manage the extensions of your project
+
+== Opening the extensions view
+The extensions view is opened from the Studio cool bar.
+
+== Adding Extensions
+From the extension view you can add the following extension to your project:
+
+* Connectors
+* Actor Filters
+* REST API Extensions
+* Themes
+* Any Java library required for your project
+
+Such extensions can be added through different channels described in the below chapters.
+
+=== From the Bonita Marketplace
+The Bonita Marketplace contains all the extensions provided and supported by Bonitasoft. It is accessible from the extension view.
+
+=== From a Maven repository
+If the extensions you are looking for are available in public or private Maven repositories, use their maven coordinates (group ID, artifact ID & Version) to add them to your project.
+
+You might need to xref:configure-maven.adoc[configure your Maven settings] to access such repositories. By default, only the Maven central repository is accessible.
+
+=== From a local file
+For extensions that aren't published in a Maven repository, you can add them from a local file on your computer. Imported extensions are added in the project internal store, an hidden folder in a Bonita project. The internal store is exported with the project (in the .bos archive), and is added to the Git repository of the project. 
+
+[CAUTION]
+====
+It is not recommended to add too many dependencies in the internal store, it makes your project heavier. This mechanism should be used to test artifacts under development, and for the artifacts that cannot be deployed on a Maven repository.
+====
+
+== Removing an extension
+Extensions can be removed from the extension view.
+
+[WARNING]
+====
+Removing an artifact might lead to errors if the artifact is used in the project. The Studio tries to detect such impacts and to alert the user before the deletion, but some specific cases cannot be detected, especially for REST API Extension usages.
+====
```

Comment:
artifact --> element

Make sure the element is not used anywhere in the project before your remove it.